scripts/install.sh

```diff
@@ -33,8 +42,12 @@ chmod 777 /opt/darshini-es/logs
 mkdir -p /opt/darshini-logs
 chmod 777 /opt/darshini-logs
 
+#install nodejs
+curl -sL https://deb.nodesource.com/setup_6.x -o nodesource_setup.sh
+sudo bash nodesource_setup.sh
+sudo apt-get install nodejs
+
 
 #get the npm modules for js files of webpages
 mkdir -p src/main/webapp/WEB-INF/node_modules
 npm install --prefix ./src/main/webapp/WEB-INF
-
```

Comment:
At the end of install.sh script, add another command with suitable comment

\# install other tools for comparison
bash script/tool-install.sh